src/components/widgets/Number.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import Header from './Header'
+import "../../styles/widget/number.css"
+import "../../styles/widget/widget.css"
+
+class Number extends Component {
+
+  calculateProgress( mainStat, goal ) {
+    const progressNum = + ( ( mainStat / goal ) * 100 ).toFixed(2)
+    return ( progressNum ).toString().concat('%')
+  }
+
+  render() {
+
+    const { title, x, y, mainStat, goal } = this.props
+    const positionStyle = { top: x, left: y }
+    const progressString = this.calculateProgress( mainStat, goal )
+    const isComplete = mainStat / goal >= 1
```

Comment:
See notes elsewhere, but please rename `mainStat` (will there be a secondary stat?). Seems like `statistic` is more meaningful, and has the added benefit of being a complete word.